tasks/configure.yml

```diff
@@ -1,4 +1,21 @@
 ---
+# logging configuration
+- name: "Configure rsyslog"
+  template:
+    src: "{{ item }}"
+    dest: "/etc/rsyslog.d/archivematica.conf"
+    backup: "yes"
+  with_first_found:
+    - files:
+        - "templates/{{ inventory_hostname }}/rsyslog.conf.j2"
+        - "templates/rsyslog.conf.j2"
+        - "etc/rsyslog.d/archivematica.conf.j2"
+  tags: "amsrc-ss-osconf"
+  notify:
+    - Restart rsyslog
+  when: "archivematica_src_syslog_enabled|bool"
+
+
```

Comment:
On CentOS rsyslog is run as root user, but this service runs as syslog user on Ubuntu. To fix the permissions issue Ubuntu needs:
```
sudo chmod 2775 /var/log/archivematica
sudo chgrp syslog /var/log/archivematica
``` 
NOTE: `/var/log/archivematica`  is 2755 by default. The `/var/log` directory has 0775 permissions and `root:syslog` as owners.

Reply:
The ```/var/log/archivematica``` permissions are set in https://github.com/artefactual-labs/ansible-archivematica-src/blob/qa/1.x/tasks/pipeline-osconf.yml#L76 , isn't that enough?

Reply:
Fixed in 546ca6b